Resources/Prototypes/_NF/tags.yml

```diff
@@ -87,3 +87,6 @@
   
 - type: Tag
   id: SpeedLoaderRifleHeavy
+
+- type: Tag
+  id: ClothingRobesGoblin
```

Comment:
Ominous.

Reply:
Even goblin clothes are capable of robbing.